scipy/stats/_distribution_infrastructure.py

```diff
@@ -0,0 +1,5082 @@
+        True
+
+        """
+        return self.moment(2, kind='central', method=method)
+
+    def standard_deviation(self, *, method=None):
+        r"""Standard deviation (square root of the second central moment)
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'transform', 'normalize', 'quadrature', 'cache'}
+            Method used to calculate the central second moment. Not
+            all methods are available for all distributions. See
+            `ContinuousDistribution.moment` for details.
+
+        See Also
+        --------
+        ContinuousDistribution.variance
+        ContinuousDistribution.mean
+        ContinuousDistribution.moment
+
+        Examples
+        --------
+        Instantiate a distribution with the desired parameters:
+
+        >>> from scipy import stats
+        >>> X = stats.Normal(mu=1., sigma=2.)
+
+        Evaluate the standard deviation:
+
+        >>> X.standard_deviation()
+        2.0
+        >>> X.standard_deviation() == X.moment(order=2, kind='central')**0.5 == X.sigma
+        True
+
+        """
+        return np.sqrt(self.variance(method=method))
+
+    def skewness(self, *, method=None):
+        r"""Skewness (standardized third moment)
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'general', 'transform', 'normalize', 'cache'}
+            Method used to calculate the standardized third moment. Not
+            all methods are available for all distributions. See
+            `ContinuousDistribution.moment` for details.
+
+        See Also
+        --------
+        ContinuousDistribution.moment
+        ContinuousDistribution.mean
+        ContinuousDistribution.variance
+
+        Examples
+        --------
+        Instantiate a distribution with the desired parameters:
+
+        >>> from scipy import stats
+        >>> X = stats.Normal(mu=1., sigma=2.)
+
+        Evaluate the skewness:
+
+        >>> X.skewness()
+        0.0
+        >>> X.skewness() == X.moment(order=3, kind='standardized')
+        True
+
+        """
+        return self.moment(3, kind='standardized', method=method)
+
+    def kurtosis(self, *, method=None, convention='non-excess'):
+        r"""Kurtosis (standardized fourth moment)
+
+        By default, this is the standardized fourth moment, also known as the
+        "non-excess" or "Pearson" kurtosis (e.g. the kurtosis of the normal
+        distribution is 3). The "Fisher" or "excess" kurtosis (the standardized
+        fourth moment minus 3) is available via the `convention` parameter.
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'general', 'transform', 'normalize', 'cache'}
+            Method used to calculate the standardized fourth moment. Not
+            all methods are available for all distributions. See
+            `ContinuousDistribution.moment` for details.
+        convention : {'non-excess', 'Pearson', 'excess', 'Fisher'}
```

Comment:
it looks odd to have two options that lead to the same result. maybe it is better to just say {'fisher', 'pearson'} or {'excess', 'non-excess'} and explain x is also known as y

Reply:
OK

Reply:
can be resolved